lib/xclarity_client/discover_request_management.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+module XClarityClient
+  class DiscoverRequestManagement < XClarityBase
+
+    include XClarityClient::ManagementMixin
+
+    def initialize(conf)
+      super(conf, DiscoverRequest::BASE_URI)
+    end
+
+    def discover_manageable_devices(ip_addresses)
+      postReq = JSON.generate(["ipAddresses":ip_addresses])
```

Comment:
Please, fix this Line and I can execute test and a better analysis of the code.

Reply:
Fixed